src/main/resources/org/jenkins_ci/plugins/run_condition/contributed/OSCondition/help.jelly

```diff
@@ -0,0 +1,27 @@
+<?jelly escape-by-default='true'?>
+<!--
+  The MIT License (MIT)
+  
+  Copyright (c) 2016 Anthony Wat
+  
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+  
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+  
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+ -->
+<j:jelly xmlns:j="jelly:core">
+	<div>Run if the current operating system is of a given type.</div>
```

Comment:
Looking at lib/runConditionPlugin/help.jelly, the run-condition plugin dynamically displays the help from each run condition including index.jelly from each run condition. So I'll keep the file structure the same for now.